src/System.Text.Json/src/System/Text/Json/Serialization/JsonNamingPolicy.cs

```diff
@@ -19,6 +19,11 @@ public abstract class JsonNamingPolicy
         /// </summary>
         public static JsonNamingPolicy CamelCase { get; } = new JsonCamelCaseNamingPolicy();
 
+        /// <summary>
+        /// Returns the naming policy for snake-casing.
+        /// </summary>
+        public static JsonNamingPolicy SnakeCase { get; } = new JsonSnakeCaseNamingPolicy();
+
         internal static JsonNamingPolicy Default { get; } = new JsonDefaultNamingPolicy();
```

Comment:
Why this is internal? Developers need to be able to use it pass through PascalCase

Reply:
A good question, but unrelated to this PR. Makes sense to open an issue so anyone would be able to find the answer.